client/views/room/contextualBar/ExportMessages/index.js

```diff
@@ -129,7 +129,7 @@ const MailExportForm = ({ onCancel, rid }) => {
 
 	const reset = useMutableCallback(() => {
 		setSelected([]);
-		$(`#chat-window-${ rid }.messages-box .message.selected`)
+		$('.messages-box .message', $(`#chat-window-${ rid }`))
```

Comment:
A simple space between them would have sufficed, but this achieves the same result. Thanks!

Reply:
I tried the same first but it didn't work, perhaps I did some errors.
Btw always welcome! 😁